cni/plugin.go

```diff
@@ -24,6 +24,11 @@ import (
 	"go.uber.org/zap"
 )
 
+var (
+	loggerName = "azure-vnet"
+	logger     = log.InitZapLogCNI(loggerName, "azure-vnet.log").With(zap.String("component", "cni"))
```

Comment:
Since for each package under cni is using the `component` as cni we can shift the `.with` part to the cni log package ? (similar to pid)